### PR TITLE
Add Utility File and Move Functions to it

### LINKED ISSUE #69: Mapper Functions Refactor
**Is your feature request related to a problem? Please describe.**
The functions that mapped over arrays to print out information were cumbersome, so we moved them to a utility file

**Describe the solution you'd like**
Moved all the map functions to a utility file

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Added utility file 
2. Moved helper functions to said file

## Purpose
Cleans up the code and makes it more readable

## Approach
Made a new utility file and exported all necessary functions where they were needed

Closes #69

### CODE REVIEW

**Review by kmausolf — COMMENTED** [review]
Great idea.  Duplicate first, then refactor like you've done here.

File: pokeapp/src/components/Sidebar.js [hunk 1]
```diff
@@ -13,41 +17,6 @@ const SideBar = ({
...
-  const weaknessIconMapper = (weaknesses) => {
-    return weaknesses?.map((weakness, index) => {
-      return (
-        <ToolTip content={weakness.name} key={index}>
-          <img className="type-icons" src={typeIconMaker(weakness.name)} />
-        </ToolTip>
-      );
-    });
-  };
-
```
Comment (gonzalespaulb) [inline pokeapp/src/components/Sidebar.js:50]: Good way to clear out the sidebar file.

File: pokeapp/src/utilities/mappers.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+export const weaknessIconMapper = (weaknesses) => {
+  return weaknesses?.map((weakness, index) => {
+    return (
+      <ToolTip content={weakness.name} key={index}>
+        <img className="type-icons" src={typeIconMaker(weakness.name)} />
+      </ToolTip>
+    );
+  });
+};
```
Comment (JayDampitan) [inline pokeapp/src/utilities/mappers.js:37]: Great job with the refactors!

**Review by gonzalespaulb — APPROVED** [review]
Adding the helper functions to the utils folder is a great way for it to be used app wide.

**Review by JayDampitan — APPROVED** [review]
Great job refactoring the mappers!

**Pull request merged**